src/ManagedServices/ManagedServices/Commands/GetAzureRmManagedServicesDefinition.cs

```diff
@@ -0,0 +1,77 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.ManagedServices.Models;
+using Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Models;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.ManagedServices.Commands
+{
+    [Cmdlet(
+    VerbsCommon.Get,
+    Microsoft.Azure.Commands.ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ManagedServicesDefinition",
+    DefaultParameterSetName = DefaultParameterSet,
+    SupportsShouldProcess = true), OutputType(typeof(PSRegistrationDefinition))]
+    public class GetAzureRmManagedServicesDefinition : ManagedServicesCmdletBase
+    {
+        protected const string DefaultParameterSet = "Default";
+
+        [Parameter(Position = 0, ParameterSetName = DefaultParameterSet, HelpMessage = "The subscription scope.")]
+        [ScopeCompleter]
+        public string Scope { get; set; }
```

Comment:
This was supposed top be removed.  Why do we need scope for a Definition?

Reply:
The user shoudlk eb able to list all defintions, or get an individual definition.  I *think* they should eb able to do both with the Id /ResourceId parameter.  Scope doesn't seem to make sense, as a definition can apply at any scope it is assgiend to.   In your cmdlet review, you indicated that this parameter was removed.

Reply:
RegistrationDefinitions are unlike Policy or Role definitions. They are a subscription level resource so they require a subscriptionid for CRUD operations. Scope, however, as I can see, is a very confusing. We are going to change this to say SubscriptionId.